builders/container-apps-internal-registry-demo/startup-script.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+set -e
+
+# Wait for the app source to be uploaded to the blob storage.
+formatted_date=$(date -u +"%a, %d %b %Y %H:%M:%S GMT")
+auth_header="Authorization: Bearer $MI_ACCESS_TOKEN"
+version_header="x-ms-version: 2019-12-12"
+date_header="x-ms-date: $formatted_date"
+file_upload_endpoint="$FILE_UPLOAD_CONTAINER_URL/$FILE_UPLOAD_BLOB_NAME"
+
+temp_app_source_dir="/tmp/appsource"
+temp_app_source_path="$temp_app_source_dir/$FILE_UPLOAD_BLOB_NAME"
+mkdir $temp_app_source_dir
+
+while [[ ! -f "$temp_app_source_path" || ! "$(file $temp_app_source_path)" =~ "compressed data" ]]
+do
+  echo "Waiting for app source to be uploaded. Please upload the app source to the endpoint specified in the Build resource's 'uploadEndpoint' property."
+  curl -H "$auth_header" -H "$version_header" -H "$date_header" -X GET "$file_upload_endpoint" -o "$temp_app_source_path" -s
+  sleep 5
+done
+
+# Extract app code to CNB_APP_DIR directory.
+echo "Found app source at '$temp_app_source_path'. Extracting to $CNB_APP_DIR"
+mkdir -p $CNB_APP_DIR
+cd $CNB_APP_DIR
+tar -xzf "$temp_app_source_path"
+
+fileCount=$(ls | wc -l)
+if [ "$fileCount" = "1" ]; then
+  # Find .jar file in the directory
+  jarfile=$(find "$CNB_APP_DIR" -maxdepth 1 -name ".[jw]ar" | head -n 1)
+
+  # unzip it if found
+  if [[ -n $jarfile ]];
+  then
+    echo "Unzip file $jarfile"
+    unzip -qq $jarfile -d $CNB_APP_DIR
+    rm $jarfile
+  fi
+fi
+
+# public cert should be in this env var
+ca_pem_decoded=$(printf "%s" "$REGISTRY_HTTP_TLS_CERTIFICATE" | base64 -d)
+echo "$ca_pem_decoded" >> /usr/local/share/ca-certificates/internalregistry.crt
```

Comment:
Do we plan on having multiple certs inside `internalregistry.crt`? If not, we probably replace the cert instead of append. Something like:
`echo "$ca_pem_decoded" > /usr/local/share/ca-certificates/internalregistry.crt`